README.md

```diff
@@ -80,7 +78,24 @@ information on changes between releases.
 
 ## Getting involved and contributing!
 
-Are you interested in contributing to kops? We the maintainers and community would love your suggestions, contributions, and help! We have a quickstart guide on [adding a feature](/docs/development/adding_a_feature.md) and the maintainers can be contacted at any time to learn more about how to get involved.
+Are you interested in contributing to kops? We the maintainers and community would love your suggestions, contributions, and help! We have a quick-start guide on [adding a feature](/docs/development/adding_a_feature.md) and the maintainers can be contacted at any time to learn more about how to get involved.
+
+In the interest of getting more new folks involved with kops, we are starting to tag issues with `good-starter-issue`. These are typically issues that have smaller scope but are good ways to start to get acquainted with the codebase.
+
+We also encourage ALL active community participants to act as if they are maintainers, even if you don't have the full permissions. This is still a small enough community that if you get involved and show an active interest and want to be a maintainer-- we would most likely love to have you. So show us what you can do!
+
+What this means:
+
+__Issues__
+* Help read and triage issues, assist when possible.
+* Tag issues that are duplicates, out of date, etc.
+  - Even if you don't have tagging permissions, make a note and tag maintainers. (`/close`,`/dupe #127`)
+
+__Pull Requests__
+* Read and review the code. Leave comments, questions, and critiques (`/lgtm` )
+* Download, compile, and run the code and make sure it does what the PR says and doesn't break something else
```

Comment:
Let's follow this PR up with some additional information in the developer section. I don't actually know what you mean by leveraging the jenkins binaries. If you can help elucidate the process, either of us can write it up and we can link to it right here.